src/assertions/text.js

```diff
@@ -0,0 +1,12 @@
+
+import should from 'should';
+
+const Assertion = should.Assertion;
+Assertion.add('text', function(string) {
+  const text = this.obj.text();
+  this.params = {
+    obj: this.obj.name(),
+    operator: `to have text '${string}' but found '${text}'`,
+  };
+  should(text.includes(string)).be.true(' ');
```

Comment:
ahhh the base-enzyme was the key that I wasnt getting. got this working right now.